### PR TITLE
Old repo

### PR DESCRIPTION
**Pull request by TillCM**
Have a look-see please ?

### CODE REVIEW

File: src/com.teamfu.be/team-reece/Models/teamfuContext.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        }
+
+        public virtual DbSet<ToDo> ToDos { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+#warning To protect potentially sensitive information in your connection string, you should move it out of source code. You can avoid scaffolding the connection string by using the Name= syntax to read it from configuration - see https://go.microsoft.com/fwlink/?linkid=2131148. For more guidance on storing connection strings, see http://go.microsoft.com/fwlink/?LinkId=723263.
+                optionsBuilder.UseSqlServer("Data Source=localhost,1433;Database=teamfu;User Id=SA;Password=Your_password123;");
```
Comment (gm7) [inline src/com.teamfu.be/team-reece/Models/teamfuContext.cs]: better to remove this - don't leave credentials in the source code  even for testing... it is so easy to leak production credentials.

File: Dockerfile [hunk 2]
```diff
@@ -6,6 +6,9 @@ WORKDIR /app
 RUN apt update
 RUN apt install tree
 
+#Environment Variables
```
Comment (gm7) [inline Dockerfile:9]: these are not needed here

**Review by gm7 — APPROVED** [review]
Looks good to me (LGTM)

File: src/com.teamfu.be/team-reece/Models/teamfuContext.cs (hunk 1 above)
Comment (TillCM) [reply src/com.teamfu.be/team-reece/Models/teamfuContext.cs]: Addressed in 0f96ffb

File: Dockerfile (hunk 2 above)
Comment (TillCM) [reply Dockerfile:9]: Addressed in 0f96ffb

**Pull request merged**